packages/@cdktf/hcl2cdk/lib/expressions.ts

```diff
@@ -0,0 +1,390 @@
+import * as t from "@babel/types";
+import { camelCase, pascalCase } from "./utils";
+import { TerraformResourceBlock } from "./types";
+
+export type Reference = {
+  start: number;
+  end: number;
+  referencee: { id: string; full: string }; // identifier for resource
```

Comment:
some examples for `id` and `full` would be nice

Reply:
Those can be found in the tests, do you think that's too hard to spot?

Reply:
No, that's reasonable 👍